docs/doc-versions.adoc

```diff
@@ -0,0 +1,9 @@
+[[doc-versions]]
+[appendix]
+== Other documentation versions
+* http://docs.janusgraph.org/0.2.1/index.html[Version 0.2.1]
+* http://docs.janusgraph.org/0.2.0/index.html[Version 0.2.0]
+* http://docs.janusgraph.org/0.1.1/index.html[Version 0.1.1]
+* http://docs.janusgraph.org/0.1.0/index.html[Version 0.1.0]
+
+This page only lists versions ${version}; and prior. Please visit the latest version to https://docs.janusgraph.org/latest/doc-versions.html[see a complete list of docs versions]."
```

Comment:
extraneous semicolon and quotation marks

Reply:
semicolon and quotation mark removed